projects/mci/test/test_mci_broadcast.c

```diff
@@ -56,19 +63,32 @@ typedef struct TestWaveSculptorBusMeasurement {
 typedef struct TestMotorControllerMeasurements {
   TestWaveSculptorBusMeasurement bus_measurements[NUM_MOTOR_CONTROLLERS];
   uint16_t vehicle_velocity[NUM_MOTOR_CONTROLLERS];
+  uint32_t status[NUM_MOTOR_CONTROLLERS];
 } TestMotorControllerMeasurements;
 
 static TestMotorControllerMeasurements s_test_measurements = { 0 };
 
 static MotorCanFrameId s_frame_id_map[] = {
   [LEFT_MOTOR_CONTROLLER * NUM_TEST_MCI_MESSAGES + TEST_MCI_VELOCITY_MESSAGE] =
-      MOTOR_CAN_LEFT_VELOCITY_MEASUREMENT_FRAME_ID,
+      LEFT_MOTOR_CONTROLLER_BASE_ADDR + WAVESCULPTOR_MEASUREMENT_ID_VELOCITY,
   [RIGHT_MOTOR_CONTROLLER * NUM_TEST_MCI_MESSAGES + TEST_MCI_VELOCITY_MESSAGE] =
-      MOTOR_CAN_RIGHT_VELOCITY_MEASUREMENT_FRAME_ID,
+      RIGHT_MOTOR_CONTROLLER_BASE_ADDR + WAVESCULPTOR_MEASUREMENT_ID_VELOCITY,
   [LEFT_MOTOR_CONTROLLER * NUM_TEST_MCI_MESSAGES + TEST_MCI_BUS_MEASUREMENT_MESSAGE] =
-      MOTOR_CAN_LEFT_BUS_MEASUREMENT_FRAME_ID,
+      LEFT_MOTOR_CONTROLLER_BASE_ADDR + WAVESCULPTOR_MEASUREMENT_ID_BUS,
   [RIGHT_MOTOR_CONTROLLER * NUM_TEST_MCI_MESSAGES + TEST_MCI_BUS_MEASUREMENT_MESSAGE] =
-      MOTOR_CAN_RIGHT_BUS_MEASUREMENT_FRAME_ID,
+      RIGHT_MOTOR_CONTROLLER_BASE_ADDR + WAVESCULPTOR_MEASUREMENT_ID_BUS,
+  [LEFT_MOTOR_CONTROLLER * NUM_TEST_MCI_MESSAGES + TEST_MCI_STATUS_MESSAGE] =
```

Comment:
Idea: stick the `X_MOTOR_CONTROLLER * NUM_TEST_MCI_MESSAGES + TEST_MCI_X_MESSAGE` expression in a macro to make this array slightly less wall-of-text like.

Reply:
Good idea, thanks, should be a lot cleaner like that